components/channel_selector_modal/channel_selector_modal.jsx

```diff
@@ -222,6 +222,7 @@ export default class ChannelSelectorModal extends React.Component {
                         buttonSubmitText={buttonSubmitText}
                         saving={false}
                         loading={this.state.loadingChannels}
+                        placeholderText={localizeMessage('multiselect.placeholder', 'Search and add members')}
```

Comment:
`Search and add members?` sounds confusing, it shouldn't be 'Search and add channels'

Reply:
This is the existing wording (as seen in this screenshot from the community server):

<img width="600" alt="Screen Shot 2019-05-09 at 10 42 23 AM" src="https://user-images.githubusercontent.com/1149597/57462638-38a39580-7247-11e9-9679-f27231c56993.png">

Reply:
I think the code here is not pointing to that modal, is the `components/channel_selector_modal/channel_selector_modal.jsx` with the title `Add Channels To **Channel Selection** List`

Reply:
You're right it's not that modal. It is the existing text for the add channels and teams to a group in system console but probably is wrong there. I'll update it.


<img width="781" alt="Screen Shot 2019-05-09 at 12 01 01 PM" src="https://user-images.githubusercontent.com/1149597/57468297-2ed35f80-7252-11e9-9979-4dbb7d043491.png">